internal/engine/experiment/dnsping/dnsping.go

```diff
@@ -119,11 +120,10 @@ func (m *Measurer) Run(
 	}
 	tk := new(TestKeys)
 	measurement.TestKeys = tk
-	mxmx := measurex.NewMeasurerWithDefaultSettings()
-	out := make(chan *measurex.DNSMeasurement)
+	out := make(chan *SinglePing)
```

Comment:
Above there's a TODO comment for me that now has been done by your PR, so please zap this comment